src/fileActions.jsx

```diff
@@ -268,45 +268,13 @@ const EditPermissionsModal = ({ selected, path }) => {
     const [errorMessage, setErrorMessage] = useState(undefined);
     const accounts = useFile("/etc/passwd", { syntax: etcPasswdSyntax });
     const groups = useFile("/etc/group", { syntax: etcGroupSyntax });
```

Comment:
As discussed, this needs to be `getent passwd` and `getent group`, but that can be a follow-up.

Reply:
I'll do it as follow up